README.md

```diff
@@ -19,10 +19,12 @@ of Node.js.
 
 It is important to remember that *other* Node.js interfaces such as
 `libuv` (included in a project via `#include <uv.h>`) are not ABI-stable across
-Node.js major versions. Thus, and addon must use N-API and/or `node-addon-api`
+Node.js major versions. Thus, an addon must use N-API and/or `node-addon-api`
 exclusively and build against a version of Node.js that includes an
 implementation of N-API (meaning a version of Node.js newer than 6.14.2) in
-order to benefit from ABI stability across Node.js major versions.
+order to benefit from ABI stability across Node.js major versions. Node.js
+provides an [ABI stability guide][] containing a detailed explanation of ABI
```

Comment:
The link does not work

Reply:
@NickNaso had to leave a blank line before the `[ABI stability guide]: <url>` line at the bottom of the file. Should work now.